packages/grid/helm/syft/templates/backend/backend-statefulset.yaml

```diff
@@ -20,7 +20,16 @@ spec:
       labels:
         {{- include "common.labels" . | nindent 8 }}
         app.kubernetes.io/component: backend
+        {{- if .Values.node.podLabels }}
+        {{- toYaml .Values.node.podLabels | nindent 8 }}
```

Comment:
When we input mutiple podsLabels , it takes the last one.

Example Values files

```
node:
    podLabels:
        test/label: 5
        test/label: 2
```

Output
```
![Uploading Screenshot 2024-04-17 at 2.59.58 PM.png…]()
```

Reply:
But if I reverse the order
![Screenshot 2024-04-17 at 3 01 58 PM](https://github.com/OpenMined/PySyft/assets/43314053/dca9ebcd-206f-4ce7-90ac-2eaef615ac41)

It takes the last one

![Screenshot 2024-04-17 at 3 02 19 PM](https://github.com/OpenMined/PySyft/assets/43314053/04188823-5adb-45fa-bbb9-27e5694ee5e2)

Reply:
It should take into account all the pod labels?

Reply:
This is valid & expected. `metadata.labels` & `metadata.annotations` are a map and not list. Both kubernetes and golang templating will hold only one value because underlying struct is a go map. You also see a squigly error line under your YAML because that's validation failure 😉

Reply:
Ahh ok, is there a way such that a user could provide multiple pod annotations to pod spec by helm values?

Reply:
Yes it is supported and this change implements it ([example](https://github.com/OpenMined/PySyft/pull/8712/files#diff-80ac10ca85a1ad75067177b222c9053f62d08a940e816bf23ebd47cf66b530fcR14-R17)), but k8s spec has [explicit directives  for the type of values](https://github.com/OpenMined/PySyft/pull/8712/files#diff-80ac10ca85a1ad75067177b222c9053f62d08a940e816bf23ebd47cf66b530fcR14-R17) the labels and annotations must have. Since keys-values must be strings. often times k8s providers will expect a JSON string as values ([GKE example](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#create_service))